new-lamassu-admin/src/components/editableProperty/EditableProperty.js

```diff
@@ -0,0 +1,80 @@
+import { makeStyles } from '@material-ui/core/styles'
+import React, { useState, memo } from 'react'
+
+import { H4, P } from 'src/components/typography'
+import { Link } from 'src/components/buttons'
+import { RadioGroup } from 'src/components/inputs'
+import { ReactComponent as EditIcon } from 'src/styling/icons/action/edit/enabled.svg'
+import { ReactComponent as EditIconDisabled } from 'src/styling/icons/action/edit/disabled.svg'
+
+import { editablePropertyStyles } from './EditableProperty.styles'
+
+const useStyles = makeStyles(editablePropertyStyles)
+
+const EditableProperty = memo(
+  ({ title, prefixText, disabled, options, value, save }) => {
+    const [editing, setEditing] = useState(false)
+    const [currentValue, setCurrentValue] = useState(value)
+
+    const classes = useStyles()
+
+    const innerSave = () => {
+      save(currentValue)
+      setEditing(false)
+    }
+
+    const innerCancel = () => setEditing(false)
+
+    // TODO: can do this or just pass the options with the right properties... which's better?
+    const radioButtonOptions = options.map(it => ({
+      label: it.display,
+      value: it.value
+    }))
```

Comment:
Our components usually take in display as instead of label, might as well make it standard.

Reply:
Okay! Today the RadioGroup uses the label property, so I'll change it to use display instead.